app/assets/javascripts/uploadcare/widget/tabs/camera-tab.coffee

```diff
@@ -60,6 +75,11 @@ uploadcare.namespace 'widget.tabs', (ns) ->
       isLocalhost = document.location.hostname == 'localhost'
       return !! @getUserMedia and Uint8Array and (isSecure or isLocalhost)
 
+    __checkCapture: () ->
+      input = document.createElement('input')
+      input.setAttribute('capture', true)
+      return !! input.capture
```

Comment:
Let's check appropriate values:

```js
input.setAttribute('capture', 'camera')
return input.capture == 'camera'
```

Reply:
if device and browser don't support capture, the `capture` attribute is undefined. if support, `capture` can be different, `true`, `camera` or whatever. So, we only need to test the existence of this

Reply:
Ok, then just

```js
input.setAttribute('capture', 'camera')
return !! input.capture
```